extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -730,6 +773,35 @@ export class AmpA4A extends AMP.BaseElement {
     });
   }
 
+  renderViaSafeFrame_() {
+    user().assert(this.creativeBody_,
```

Comment:
You're right.  I copied this from the existing `renderViaCrossDomainIframe_`.  Fixed in both places.